documentation/modules/using-must-gather.adoc

```diff
@@ -30,11 +30,38 @@ $ oc adm must-gather --image={must-gather}
 +
 The data is saved as `/must-gather/must-gather.tar.gz`. You can upload this file to a support case on the link:https://access.redhat.com/[Red Hat Customer Portal].
 
-* To gather data for the past 24 hours:
+. Optional: Run the `oc adm must-gather` command with the following options to gather filtered data:
+
+* Namespace:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image={must-gather} \
+  -- NS=<namespace> /usr/bin/targeted
+----
+
+* Migration plan:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image={must-gather} \
+  -- PLAN=<migration_plan> /usr/bin/targeted
+----
+
+* Virtual machine:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image={must-gather} \
+  -- VM=<vm_id> NS=<namespace> /usr/bin/targeted
```

Comment:
Just for better understading what VM ID it is (not an ID from vmware/rhv/openshift/..), the VM ID is the one from the Plan CR